ethcontract-generate/src/generate.rs

```diff
@@ -1,5 +1,3 @@
-#![deny(missing_docs)]
```

Comment:
Can we put this back before merging?

Reply:
I've made this module public, so this annotation is now redundant: there's one like this in the root `lib.rs` file.

Reply:
Oh shoot! :see_no_evil: I missed that.